app/services/prisoner_slot_availability.rb

```diff
@@ -74,4 +91,10 @@ def end_date
     # ensures the range does not go over the 28 days constraint
     @end_date ||= [@end_date, start_date + 28.days].min
   end
+
+  def in_trial?
```

Comment:
The naming of this method, and class I find very confusing.

We have the class named `PrisonerSlotAvailability` but with this change is no longer just does prisoner availability.

Then the name of this method is generic so made me thing it was related to prisoner availability to it does prison availability.

We have a class called `ApiSlotAvailability`, I think that is used on the deprecated endpoint, that could be a good name for this class?